Gruntfile.js

```diff
@@ -19,9 +19,48 @@ module.exports = exports = function(grunt) {
                  '    (c) 2013-2015 Mozilla, Apache License 2.0\n' +
                  '*/\n';
 
+    var babelModuleIdProvider = function getModuleId(moduleName) {
+        var files = {
+            'src/localforage': 'localforage',
+            'src/utils/serializer': 'localforageSerializer',
+            'src/drivers/indexeddb': 'asyncStorage',
+            'src/drivers/localstorage': 'localStorageWrapper',
+            'src/drivers/websql': 'webSQLStorage'
+        };
+
+        return files[moduleName] || moduleName.replace('src/', '');
+    };
+
     grunt.initConfig({
+        babel: {
+            options: {
+                loose: 'all',
+                modules: 'umd',
+                moduleIds: true,
+                // sourceMap: true,
+                getModuleId: babelModuleIdProvider,
+                plugins: ['system-import-transformer']
+            },
+            dist: {
+                files: {
+                    'dist/es5src/localforage.js': 'src/localforage.js',
```

Comment:
Why do we need to have an `es5src` folder of the individual components? I don't think this needs to be more than a build artifact. I would rather reduce the ways people can include `localforage.js`, so I don't mind if this means `require` people need to just import the whole file. I think that's easier to support.